packages/scene-composer/src/components/panels/scene-settings/MatterportTagSync.tsx

```diff
@@ -135,6 +132,8 @@ export const MatterportTagSync: React.FC = () => {
     for (const key in oldTagMap) {
       handleRemoveMatterportTag(oldTagMap[key].nodeRef);
     }
+
+    setSyncTagStatus('success');
```

Comment:
I think you have to check here response coming from prepareWorkspace then set the state to success. It could be still loading too even though the response is successful.

Reply:
this is the end of the entire sync function. all tags have been created already at this point. if any API failed before, it shouldn't reach here. `UPDATING` vs `ACTIVE` is not something is function needs to worry about.

I'll update the catch to cover everything above this line.

Reply:
What is the assumed state for success? Even though tags are in updating state?